app/assets/stylesheets/comments.scss

```diff
@@ -143,3 +143,7 @@
   padding: 0.7em;
   display: none;
 }
+
+.failed-validation {
+  border: 2px solid #d21919 !important;
+}
```

Comment:
Sorry, I don't think I explained clearly enough in my previous comment. You only need to add new styles to _one_ file. Your style will be available everywhere regardless of what controller it's under.